### DISCUSSION TITLE
Section 3.3: Add outreachy

### DISCUSSION THREAD

**Discussion by hamogu** [astropy/astropy-v5.0-paper#82] [Text comments] [answered]
We also used Moore money to fund one outreachy participant (https://github.com/astropy/astropy-project/blob/90fc725d115c96b510b9fa2b012a5844f73a1ba8/finance/proposal-calls/2020-IDEA/IDEA_2020_outreachy.md). The financial commitment for that is comparable to WoCC, so it should also be mentioned even if it might not get a full subsection. People to contact for more details are ##Team [Brigitta Sipőcz](https://github.com/bsipocz) [David Pérez-Suárez](https://github.com/dpshelio) [Stuart Mumford](https://github.com/Cadair) [Zé Vinícius](https://github.com/mirca)

**Comment by pllim** [comment]
We did reach out to a few people to write this part but with no success. Contribution is welcome!

**Reply by hamogu** [reply]
Unfortunately, I don't know enough about it to write that either.

**Reply by adrn** [reply]
We would love to have some text on this, and even had a placeholder section on this when we solicited authors. I don't know anything about the program, so we would need @dpshelio @cadair or @mirca to add a few sentences. Would any of you be up for it? (Brigitta already declined)

**Reply by hamogu** [reply]
@bsipocz Can you write a few sentences to point to material that the paper team can use to write a outreachy? Or tag that person who got the outreachy grant so we can ask them to write a few sentences? (I don't know the GH handle for them.)

**Reply by mirca** [reply]
Hi Adrian and Moritz, I'm not familiar with Outreachy or the extent to which AstroPy participated in it.

**Reply by Cadair** [reply]
Sorry for the slow response, but I really can't remember enough about the astropy side of it to contribute anything meaningful

**Reply by dpshelio** [reply]
Same on my side. I know how outreachy works, but I was not even aware that astropy took part on it! 😱

**Comment by adrn** [comment, marked as answer]
Unfortunately, with no one who coordinated the Outreachy program able to write text, we won't be able to add this to the paper.
